tests/skipjob/conditions/01-assert.yaml

```diff
@@ -0,0 +1,40 @@
+#apiVersion: kuttl.dev/v1beta1
+#kind: TestAssert
+#timeout: 60
```

Comment:
Commented out, TODO?

Reply:
See PR description:

Testing of conditions does not work on emulated control plane, so these tests are commented out. Maybe setup kind cluster in workflow? Should probably be a separate PR